tracks/csharp/exercises/nucleotide-count/mentoring.md

```diff
@@ -64,6 +64,9 @@ public static class NucleotideCount
 
 ### Common Suggestions
 
- * Iterating over the input string more than once wastes cycles, memory, etc. Often it isn't obvious that this is happening because methods like `ToList`, `GroupBy` or `Count` will iterate a list without the student having to write a loop.
-   * Hence, using `sequence.Any()` or `sequence.Where(...)` to implement the exception is wrong. It should be thrown within the single iteration.
-   * Similarly, though there are numerous ways to use LINQ to solve this problem it should be avoided. All of them involve iterating the input string multiple times.
\ No newline at end of file
+As the README doesn't explicitly mention that a student should optimize for performance, don't discard solutions just on this ground. Rather, use it as a chance to start a discussion about readability/performance trade-offs and when to prefer which.
+
+* If you look for a performance-oriented solution, iterating over the input string more than once wastes cycles, memory, etc. Often, it isn't obvious that this is happening because methods like `ToList`, `GroupBy` and `Count` will iterate over all the string's characters without the student having to write a loop.
```

Comment:
```suggestion
* If you look for a performance-oriented solution, iterating over the input string more than once wastes cycles, memory, etc. Often, it isn't obvious that this is happening because LINQ's `ToList`, `GroupBy` and `Count` methods will iterate over _all_ the `string`'s characters.
```